mmdet/core/hook/sync_random_size_hook.py

```diff
@@ -0,0 +1,53 @@
+import random
+
+import torch
+from mmcv.runner import get_dist_info
+from mmcv.runner.hooks import HOOKS, Hook
+from torch import distributed as dist
+
+
+@HOOKS.register_module()
+class SyncRandomSizeHook(Hook):
+    """Change and synchronize the random image size across ranks, currently
+    used in YOLOX.
+
+    Args:
+        ratio_range (tuple[int]): Random ratio range. It will be multiplied
+            by 32, and then change the dataset output image size.
+            Defaults to (14, 26).
+        img_scale (tuple[int]): input image size. Defaults to (640, 640).
+        interval (int): The interval of change image size. Default to 10.
+    """
+
+    def __init__(self,
+                 ratio_range=(14, 26),
+                 img_scale=(640, 640),
+                 interval=10):
+        self.rank, world_size = get_dist_info()
```

Comment:
If `world_size` is not used, replace with `_`

Reply:
`world_size`  is used in the next line )